app/src/main/java/eu/kanade/tachiyomi/ui/manga/MangaScreenModel.kt

```diff
@@ -718,6 +721,23 @@ class MangaScreenModel(
                 read = read,
                 chapters = chapters.toTypedArray(),
             )
+
+            if (read) {
+                val maxChapterNumber = chapters.maxOf { it.chapterNumber }
+
+                val formattedChapterNumber = maxChapterNumber.toInt()
+
+                val result = snackbarHostState.showSnackbar(
+                    message = context.stringResource(MR.strings.confirm_tracker_update, formattedChapterNumber),
+                    actionLabel = context.stringResource(MR.strings.action_ok),
+                    duration = SnackbarDuration.Short,
```

Comment:
Should probably show until dismissed?

Reply:
In my opinion no, I tried it and it just felt really annoying, that's why I added it. I feel like there are people who won't like this setting and we don't give them an option to disable it so the least we can do is put it on a timer. And the short duration is actually decently long.

Reply:
> I feel like there are people who won't like this setting and we don't give them an option to disable it so the least we can do is put it on a timer.

Voicing the presence of such users here too o/
No need to be conservative with the "feel like" either, you asked if that would be the case and we answered xD